frontend/public/components/start-guide.tsx

```diff
@@ -72,7 +70,7 @@ export const withStartGuide = (WrappedComponent, disable: boolean = true) =>
           </div>
           {// Whitelist certain resource kinds that should not be disabled when no projects are
           // available. Disabling should also be optional
-          !disable || WHITELIST.includes(kind) ? (
+          !disable || !kindObj.namespaced || kind === RoleModel.kind ? (
```

Comment:
It might make sense to include the RoleBindings as well here. We weren't before, but it seems like it's very similar to the Roles list page in that RBs and CRBs are listed together on the same page. An edge case might be possible where a user can create CRBs without having access to any namespaces. We also might need to look into any other possible cases like this where namespaced and non-namespaced resources are listed together on the same page. We probably should only disable pages that exclusively list namespaced resources. @spadgett any thoughts?